hibernate-vector/src/main/java/org/hibernate/vector/OracleVectorJdbcType.java

```diff
@@ -0,0 +1,186 @@
+package org.hibernate.vector;
```

Comment:
The classes OracleVectorJdbcType.java, OracleByteVectorJdbcType.java, OracleFloatVectorJdbcType.java and OracleDoubleVectorJdbcType.java share a big part of the code. Can this shared code be added to another class (maybe an abstract class, or a generic class) that will be extended by these classes? Just to avoid having several copies of the same lines of code